.github/workflows/prerelease.yml

```diff
@@ -85,9 +85,8 @@ jobs:
           flavor: |
             latest=false
           tags: |
-            type=semver,pattern={{version}}+py${{ matrix.python-version }}
+            type=match,pattern=v(.*),group=1,suffix=-py${{ matrix.python-version }}
             type=semver,pattern={{version}},enable=${{ matrix.python-version == 3.8 }}
```

Comment:
This works as-is because it's not of a Python-specific format (`1.2.3-alpha.1-py3.8`) and only applied to the "default" Python version